packages/providers/email-amazon-ses/package.json

```diff
@@ -42,5 +42,8 @@
   "engines": {
     "node": ">=12.x.x <=16.x.x",
     "npm": ">=6.0.0"
+  },
+  "strapi": {
+    "isProvider": true
```

Comment:
Actually we don't use this key at all. I suggest to remove it for the moment.
The current logic is 
```
strapi: {
   kind: 'plugin'
}
```
We could use the same for providers: `kind: 'provider'` but that would need a bit of thinking first. `provider` or `email-provider` or else?

I propose to remove the key from all packages. wdyt @alexandrebodin ?